Regression/Core/src/main/java/org/tribuo/regression/example/GaussianDataSource.java

```diff
@@ -54,10 +54,10 @@
     private int numSamples;
 
     @Config
-    private float slope;
+    private float slope = 0.0f;
```

Comment:
0 always has the right type, unless I miss my guess. 0.0f seems like gilding the lily here.

Reply:
My IDE complains and it doesn't compile without it.

Reply:
I also added description fields to all the config variables while I was in here.